assets/files/etc/haproxy/haproxy.cfg.template

```diff
@@ -18,5 +18,5 @@ backend masters
    option external-check
    option log-health-checks
    external-check path "/usr/bin:/usr/local/bin"
-   external-check command "/usr/local/etc/haproxy/members_healthcheck.sh"
-   balance     roundrobin
\ No newline at end of file
+   external-check command "/etc/haproxy/members_healthcheck.sh"
```

Comment:
Checked it locally on my setup looks OK.
You should leave the external check command point to pod's directory, i.e
"/usr/local/etc/haproxy/members_healthcheck.sh"

With your change, we get [1] Error

[1] [ALERT] 078/120525 (36) : Failed to exec process for external health check: No such file or directory. Aborting.